recipe/meta.yaml

```diff
@@ -32,9 +32,7 @@ requirements:
     - python
     - pillow >=10.0.1
     - deprecated
-    # TODO: Remove when `qpdf` version `12` is added.
-    # xref: https://github.com/conda-forge/pikepdf-feedstock/issues/124
-    - {{ pin_compatible("qpdf") }}
+    - qpdf
```

Comment:
We can drop `qpdf` from `requirements/run`. Having it in `requirements/host` should now be sufficient thanks to `run_exports`

```suggestion
```